audio_sync/cli.py

```diff
@@ -142,6 +181,18 @@ def _GetWaveDurationSecs(wav_path):
     wav.close()
 
 
+def _GetStartTime(start_time_str):
+  hh_mm_ss = start_time_str.split(':')
```

Comment:
Use instead datetime.datetime.strptime(start_time_str, "%H:%M:%S"). It'll raise a ValueError if the string has the incorrect format. Notice this will accept strings like "1:2:3" (which the current code accepts anyway).

Reply:
Done

Reply:
_GetStartTime() stills needs to be deleted.

Reply:
Odd. I remember deleting it. Done now, I hope :-)